_posts/2022-06-22-howto-use-the-function-builder-api.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Create functions from source code with the OpenFaaS Function Builder API"
+description: "We show you how to build function images with the OpenFaaS Function Builder API"
+date: 2022-06-22
+image: /images/2022-06-create-funcitons-from-source-with-pro-builder-api/background.png
+categories:
+- containers
+- functions
+- kubernetes
+- oci
+author_staff_member: han
+---
+
+We show you how to build function images with the OpenFaaS Function Builder API
+
```

Comment:
Here, introduce the use-case for why you may want to build your own images.

Who are these people? 

Why would they want to do this?

Reply:
I just pushed some changes that add this.